platforms/documentation/docs/src/docs/userguide/dep-man/dependency-resolution/artifact_transforms.adoc

```diff
@@ -49,25 +52,27 @@ image::artifact-transform-3.png[]
 1. **Implement a Transform**: You define an artifact transform by creating a class that implements the link:{javadocPath}/org/gradle/api/artifacts/transform/TransformAction.html[`TransformAction`] interface.
 This class specifies how the input artifact should be transformed into the output artifact.
 
-2. **Declare request Attributes**: Attributes (key-value pairs used to describe different variants of a component) like `org.gradle.usage=java-api` and `org.gradle.usage=java-runtime` are used to specify the desired artifact format/type.
+2. **Declare request Attributes**: Attributes (key-value pairs used to describe different variants of a component) like `org.gradle.usage=java-api` and `org.gradle.usage=java-runtime` are used to specify the desired artifact format or type.
 
-3. **Register a Transform**: You register the transform in your build script using the link:{javadocPath}/org/gradle/api/artifacts/dsl/DependencyHandler.html#registerTransform(java.lang.Class,org.gradle.api.Action)[`registerTransform()`] method of the link:{javadocPath}/org/gradle/api/Project.html#dependencies(groovy.lang.Closure)[`dependencies`] block.
-This method links the input attributes to the output attributes and associates them with the transform action class.
+3. **Register a Transform**: You register the transform by using the link:{javadocPath}/org/gradle/api/artifacts/dsl/DependencyHandler.html#registerTransform(java.lang.Class,org.gradle.api.Action)[`registerTransform()`] method of the link:{javadocPath}/org/gradle/api/Project.html#dependencies(groovy.lang.Closure)[`dependencies`] block.
+This method tells Gradle that a transform can be used to modify the artifacts of any variant that possesses the given "from" attributes.
+It also tells Gradle what new set of "to" attributes will describe the format or type of the resulting artifacts.
 
-4. **Use the Transformed Artifacts**: When a resolution requires an artifact matching the transform's output attributes, Gradle automatically applies the registered transform to the input artifact and provides the transformed artifact as a result.
+4. **Use the Transform**: When a resolution requires a variant that isn't already present in the selected component (because none of the actual variants possess compatible attributes to the requested attributes), Gradle doesn't just give up!
```

Comment:
```suggestion
4. **Use the Transform**: When a resolution requires an artifact that isn't already present in the selected component (because none of the actual artifact possess compatible attributes to the requested attributes), Gradle doesn't just give up!
```